framework/mesh/mesh_generator/split_file_mesh_generator.cc

```diff
@@ -40,8 +40,9 @@ SplitFileMeshGenerator::GetInputParameters()
     "split_mesh",
     "Path of the directory to be created for containing the split meshes.");
 
-  params.AddOptionalParameter(
-    "split_file_prefix", "split_mesh", "Prefix to use for all split mesh files");
+  params.AddOptionalParameter("split_file_prefix",
```

Comment:
I would recommend changing this to just `file_prefix` since this is always within the block for creating s split file mesh.